src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const populationArr = [...population]
+  .map(el => Number(el.textContent.split(',').join('')));
+
+const total = populationArr.reduce((sum, cur) => sum + cur);
```

Comment:
The initial value is optional in the reduce method, but it's better to always add it to make the function more understandable 
```suggestion
const total = populationArr.reduce((sum, cur) => sum + cur, 0);
```

Reply:
- [DEMO LINK](https://Yevhen-Ivanchuk.github.io/js_get_data_DOM/)